### PR TITLE
Protostream version 5.0.8.final manual upgrade

### PR DESCRIPTION
**Pull request by apupier**
# Description

not sure that it is the direction we want to go but at least it allows to see which modifications are needed to integrate protostream 5.0.8

# Target

- [ ] I checked that the commit is targeting the correct branch (note that Camel 3 uses `camel-3.x`, whereas Camel 4 uses the `main` branch)

# Tracking
- [ ] If this is a large change, bug fix, or code improvement, I checked there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).

# Apache Camel coding standards and style

- [ ] I checked that each commit in the pull request has a meaningful subject line and body.

- [ ] I have run `mvn clean install -DskipTests` locally from root folder and I have committed all auto-generated changes.

### CODE REVIEW

File: components/camel-infinispan/camel-infinispan/pom.xml [hunk 1]
```diff
@@ -158,6 +168,21 @@
...
+                <artifactId>protostream</artifactId>
+                <version>${protostream-version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.infinispan.protostream</groupId>
+                <artifactId>protostream-types</artifactId>
+                <version>${protostream-version}</version>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
```
Comment (apupier) [inline components/camel-infinispan/camel-infinispan/pom.xml:184]: Just this be placed in parent pom?

File: components/camel-infinispan/camel-infinispan/pom.xml [hunk 2]
```diff
@@ -64,6 +64,16 @@
...
+        <dependency>
+            <groupId>org.infinispan.protostream</groupId>
+            <artifactId>protostream</artifactId>
+            <version>${protostream-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.infinispan.protostream</groupId>
+            <artifactId>protostream-types</artifactId>
+            <version>${protostream-version}</version>
+        </dependency>
```
Comment (apupier) [inline components/camel-infinispan/camel-infinispan/pom.xml:76]: realizing I left direct dependecy and dependency mangement, will remove one of them

**Review by oscerd — APPROVED** [review]

**Comment by apupier** [comment]
I do not undertsand why there is a lot of uncommitted changes with the modified code in this PR

**Review by davsclaus — APPROVED** [review]

**Comment by apupier** [comment]
the force-push is only a rebase to try to have a green build

**Pull request merged**